packages/truffle-config/index.js

```diff
@@ -77,7 +75,17 @@ function Config(truffle_directory, working_directory, network) {
     // These are already set.
     truffle_directory: function() {},
     working_directory: function() {},
-    network: function() {},
+    network: {
+      get: function() {
+        return this._values.network;
+      },
+      set: function(val) {
```

Comment:
I don't want to really make a thing out of this, but I just cringe slightly when I see a variable name as bland as `val`...and that it is shortened.  I don't know if I have a better suggestion.

Reply:
Any thoughts?

Reply:
We can change these later